code/modules/mob/living/carbon/carbon.dm

```diff
@@ -588,14 +593,20 @@
 	if(!GLOB.tinted_weldhelh)
 		return
 	tinttotal = get_total_tint()
+	clear_fullscreen("tint", 0)
 	if(tinttotal >= TINT_BLIND)
 		become_blind(EYES_COVERED)
-	else if(tinttotal >= TINT_DARKENED)
+	else if(tinttotal >= TINT_DARKENED && tinttotal < TINT_BLIND)
```

Comment:
```suggestion
	else if(tinttotal >= TINT_DARKENED)
```
anything with a tinttotal that WASN'T < TINT_BLIND would have been caught by the if(tinttotal >= TINT_BLIND) check above and wouldn't get to this point